client/state/data-layer/wpcom/account-recovery/reset/test/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+
+import { handleResetPasswordRequest } from '../';
+import useNock from 'test/helpers/use-nock';
+
+import {
+	ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_SUCCESS,
+	ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_ERROR,
+} from 'state/action-types';
+
+describe( 'handleResetPasswordRequest()', () => {
+	const dispatch = sinon.spy();
+
+	const apiBaseUrl = 'https://public-api.wordpress.com:443';
+	const endpoint = '/wpcom/v2/account-recovery/reset';
+
+	const params = {
+		userData: {
+			user: 'foo',
+		},
```

Comment:
The comment in the code under test says:

```js
// userData can be either { user } or { firstname, lastname, url }
```

Would a test where `userData` contains `{ firstname, lastname, url }` be useful here?